app-misc/nnn/nnn-4.7.ebuild

```
@@ -0,0 +1,96 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit bash-completion-r1 flag-o-matic toolchain-funcs xdg-utils
+
+DESCRIPTION="The missing terminal file browser for X"
+HOMEPAGE="https://github.com/jarun/nnn"
+SRC_URI="https://github.com/jarun/nnn/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+IUSE="+readline pcre 8contexts icons nerdfonts emoji qsort colemak gitstatus namefirst restorepreview"
+
+DEPEND="sys-libs/ncurses:=
+	pcre? ( dev-libs/libpcre )
+	readline? ( sys-libs/readline:= )
+	elibc_musl? ( sys-libs/fts-standalone )"
+BDEPEND="virtual/pkgconfig"
+RDEPEND="${DEPEND}"
+REQUIRED_USE="?? ( icons nerdfonts emoji )"
+
+src_prepare() {
+	default
+	tc-export CC
+	use elibc_musl && append-flags "-lfts"
+	# When one of nnn's bundled patches is used, the Makefile patches the
+	# source, compiles and then reverts the patch. This causes the source files
+	# to be newer than the compiled binary so that it get's compiled again for
+	# the 'install' target. Removing the dependency on 'all' fixes this.
```

Typically, the convention is to refer to the bug report instead.

```suggestion
	# See bug: https://github.com/jarun/nnn/issues/1493
```

---

Of course. I wrote that before I found the issue.